Maestro/subscriptions.json

```diff
@@ -241,6 +276,32 @@
         }
       }
     },
+    // Update dependencies in CoreFX release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest.txt",
```

Comment:
While I agree we likely want these other triggers. I'm not sure we want them coming from their master branches.  In fact for now I think I would leave them out because I don't want non-release-2.0.0 packages coming into the release\2.0.0 branches.